ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py

```diff
@@ -0,0 +1,56 @@
+from ADR.Components.Aerodynamic_components.Aerodynamic_surface import Aerodynamic_surface
+import numpy.testing as npt
+import pytest
+
+
+@pytest.fixture
+def aerodynamic_surface():
+    aerodynamic_surface_parameters = {"airfoil_clmax": 2.5, "airfoil1_name": "s1223", "airfoil2_name": "s1223",
+                                      "airfoil3_name": "s1223", "span1": 2, "span2": 1.5, "chord1": 0.40, "chord2": 0.30,
+                                      "chord3": 0.35, "twist1": 0, "twist2": 0, "twist3": 0, "incidence": 0, "x": 1, "z": 1}
+    aerodynamic_surface = Aerodynamic_surface(aerodynamic_surface_parameters)
+    return aerodynamic_surface
+
+
+def test_update_alpha(aerodynamic_surface):
+    aerodynamic_surface.update_alpha(20)
+    new_alpha = aerodynamic_surface.attack_angle
+    npt.assert_almost_equal(new_alpha, 20)
+
+
+def test_get_alpha_range(aerodynamic_surface):
+    x = [-10, -9, -8, -7, -6, -5, -4, -3,
+         -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
You can change it for:

```
x = list(range(-10, 11))
```

If you want to know more about the range iterator, take a look [here](https://stackoverflow.com/questions/18265935/python-create-list-with-numbers-between-2-values).